torch/distributed/algorithms/_checkpoint/checkpoint_wrapper.py

```diff
@@ -124,7 +143,10 @@ def checkpoint_wrapper(
             The module to be wrapped
         checkpoint_impl (Optional[CheckpointImpl]):
             The checkpointing implementation to use. Currently only
-            CheckpointImpl.REENTRANT is supported.
+            CheckpointImpl.REENTRANT is supported. Note that this will only
+            be passed into the `torch.utils.checkpoint.checkpoint`
+            implementation, and is ignored if a custom ``checkpoint_fn`` is
+            specified.
         offload_to_cpu (Optional[bool]):
             Whether to offload outer activations to CPU. Note that this
             currently only works with CheckpointImpl.REENTRANT.
```

Comment:
Should we add docs for the new arguments `checkpoint_fn`, `checkpoint_fn_args`, and `checkpoint_fn_kwargs`?

Reply:
Yes, great catch!